<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QyController" name="dial_test_middle_sym"/>
     </item>
     <item>
      <widget class="QyController" name="dial_test_middle"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QyController" name="dial_test_little"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QyController" name="dial_test_big">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>382</width>
      <height>282</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>526</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QyController" name="controller_mid_1">
       <property name="caption">
        <string>cap short</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QyController" name="controller_mid_2">
       <property name="caption">
        <string>cap long cap long</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QyController" name="controller_mid_3">
       <property name="graphicStyle">
        <enum>Qy::GSI_HalfRotary</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QyController" name="controller_mid_4"/>
     </item>
     <item>
      <widget class="QyController" name="controller_mid_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QyController" name="controller">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>331</width>
       <height>100</height>
      </rect>
     </property>
     <property name="graphicStyle">
      <enum>Qy::GSI_HalfRotary</enum>
     </property>
     <property name="caption">
      <string>asdkaslkdjaslkdjaslkdja</string>
     </property>
     <property name="suffix">
      <string>km/h</string>
     </property>
    </widget>
   </widget>
   <widget class="QyButton" name="button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>test</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QyController</class>
   <extends>QFrame</extends>
   <header>QyController.h</header>
  </customwidget>
  <customwidget>
   <class>QyButton</class>
   <extends>QWidget</extends>
   <header>QyButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
